### PR TITLE
Ashlesha PR T2.1 Gif and Readme Updates

### PR DESCRIPTION
**Pull request by Mohammed-MSFT**

### CODE REVIEW

File: samples/msgext-message-reminder/csharp/MessagingExtensionReminder/Images/4.teamscope.png
Comment (Pawank-MSFT) [inline samples/msgext-message-reminder/csharp/MessagingExtensionReminder/Images/4.teamscope.png:1]: Please hide the member names in the image.

File: samples/msgext-message-reminder/csharp/MessagingExtensionReminder/Images/6.task-created-success-new.png
Comment (Pawank-MSFT) [inline samples/msgext-message-reminder/csharp/MessagingExtensionReminder/Images/6.task-created-success-new.png:1]: Please hide the member names in the image.

File: samples/msgext-message-reminder/csharp/MessagingExtensionReminder/Images/7.Reminder-card-of-task-at-scheduled-date-and-time.png
Comment (Pawank-MSFT) [inline samples/msgext-message-reminder/csharp/MessagingExtensionReminder/Images/7.Reminder-card-of-task-at-scheduled-date-and-time.png:1]: Please hide the member names in the image.

**Review by Pawank-MSFT — COMMENTED** [review]
given small comments

**Review by Pawank-MSFT — APPROVED** [review]
looks correct, Approving!

**Comment by Pawank-MSFT** [comment]
Executed Build Pipeline:
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/10236640020

**Review by Harikrishnan-MSFT — APPROVED** [review]
Approving!

**Pull request merged**